module/zfs/dsl_scan.c

```diff
@@ -2966,6 +3026,26 @@ dsl_scan_active(dsl_scan_t *scn)
 	return (used != 0);
 }
 
+static boolean_t
+dsl_scan_check_deferred_vdevs(vdev_t *vd)
```

Comment:
This seems similar to dsl_scan_reset_deferred(), so a similar name would be `dsl_scan_check_deferred()` (or add _vdevs to both?)

You might also put the two functions next to each other in the file.